tool/tctl/common/decision/evaluate_ssh_command.go

```diff
@@ -30,25 +30,20 @@ import (
 // EvaluateSSHCommand is a command to evaluate
 // SSH access via the Teleport Decision Service.
 type EvaluateSSHCommand struct {
-	output io.Writer
-
-	sshDetails sshDetails
-	command    *kingpin.CmdClause
-}
-
-type sshDetails struct {
-	serverID string
-	username string
-	login    string
+	Output   io.Writer
+	Username string
+	ServerID string
+	Login    string
+	command  *kingpin.CmdClause
 }
 
 // Initialize sets up the "tctl decision evaluate ssh" command.
 func (c *EvaluateSSHCommand) Initialize(cmd *kingpin.CmdClause, output io.Writer) {
-	c.output = output
+	c.Output = output
 	c.command = cmd.Command("evaluate-ssh-access", "Evaluate SSH access for a user.").Hidden()
-	c.command.Flag("username", "The username to evaluate access for.").StringVar(&c.sshDetails.username)
-	c.command.Flag("login", "The os login to evaluate access for.").StringVar(&c.sshDetails.login)
-	c.command.Flag("server-id", "The host id of the target server.").StringVar(&c.sshDetails.serverID)
+	c.command.Flag("username", "The username to evaluate access for.").StringVar(&c.Username)
```

Comment:
would it make sense to make this required at the parser level, or is there some advantage to the empty check in `Run()`?

```suggestion
	c.command.Flag("username", "The username to evaluate access for.").Required().StringVar(&c.Username)
```